src/RequestProcessor.cs

```diff
@@ -355,6 +356,11 @@ internal StreamingMessage ProcessFunctionEnvironmentReloadRequest(StreamingMessa
 
             var rpcLogger = new RpcLogger(_msgStream);
             rpcLogger.SetContext(request.RequestId, null);
+
+            var setCurrentDirMessage = string.Format(PowerShellWorkerStrings.SettingCurrentDirectory, environmentReloadRequest.FunctionAppDirectory);
+            rpcLogger.Log(isUserOnlyLog: false, LogLevel.Trace, setCurrentDirMessage);
+            Directory.SetCurrentDirectory(environmentReloadRequest.FunctionAppDirectory);
```

Comment:
Should we check that the directory exists before setting it? I remember that in the past for proxy call the working directory could be null.

Reply:
Checking for null: yes, thank you, done.
Checking that the directory exists: I don't know if we have a better way of handling the situation when it does not exist. Perhaps the best we can do is trust the host and fail if it does not exist - this is what NodeJs and Python workers also do.

Reply:
Yes. Trusting the host is OK. Any error handling should be done in  the host for this.